docs/advanced/document-store/index.md

```diff
@@ -133,6 +134,203 @@ da = DocumentArray(
 
 Using dataclass gives you better type-checking in IDE but requires an extra import; using dict is more flexible but can be error-prone. You can choose the style that fits best to your context.
 
+
+## Search summary
+
+DocArray supports multiple storage backends with different search features. The following table showcases relevant functionalities that are supported (✅) or not supported (❌) in DocArray depending on the backend:
+
+
+| Name                                                           | Construction                             | vector search | vector search + filter | filter|
+|----------------------------------------------------------------|------------------------------------------|---------------|------------------------|---------------|
+| [`None`](./../../../fundamentals/documentarray/#documentarray) | `DocumentArray()`                        | ✅             | ✅                      | ✅             |
+| [`Sqlite`](./sqlite/#sqlite)                                   | `DocumentArray(storage='sqlite')`        | ❌             | ❌                      | ✅             | 
+| [`Weaviate`](./weaviate/#weaviate)                             | `DocumentArray(storage='weaviate')`      | ✅             | ✅                      | ✅             |
+| [`Qdrant`](./qdrant/#qdrant)                                   | `DocumentArray(storage='qdrant')`        | ✅             | ✅                      | ❌             |
+| [`Annlite`](./annlite/#annlite)                                | `DocumentArray(storage='anlite')`        | ✅             | ✅                      | ✅             |
+| [`ElasticSearch`](./elasticsearch/#elasticsearch)              | `DocumentArray(storage='elasticsearch')` | ✅             | ✅                      | ✅             |
+
+
+Here we understand by 
+
+- **vector search**: perform approximate nearest neighbour search (or exact full scan search). The input of the search function is a numpy array or a DocumentArray containing an embedding. 
```

Comment:
```suggestion
- **fast vector search**: perform approximate nearest neighbour search (or exact full scan search). The input of the search function is a numpy array or a DocumentArray containing an embedding. 
```

Reply:
won't add 'fast' because it depends on the DocumentArray that you use!